devproject/test_settings.py

```diff
@@ -56,3 +56,15 @@
     }
 
     TEST_NAME = "travis_ci_test"
+
+AUTH_PASSWORD_VALIDATORS = [
```

Comment:
```suggestion
# Simplify password validation to ease writing test assertions
AUTH_PASSWORD_VALIDATORS = [
```

Reply:
Can you move this setting near the `PASSWORD_HASHERS`?

Reply:
Done